### PR TITLE
Update kit-assembly.md

### PR DESCRIPTION
**Pull request by BitBait**
Updated kti ameebly for 2018 
-Changed the hardware list into 3 different parts, changes made to steps 4,7,11
-Added 2mm screwdriver to list of needed things

### CODE REVIEW

File: content/tutorials/kit-assembly.md [hunk 1]
```diff
@@ -2,27 +2,34 @@
 title: Connecting your kit
 ---
 
-## The bits you should have
+##Essential Hardware##
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: Don't need the hashes on the end of this line

File: content/tutorials/kit-assembly.md [hunk 2]
```diff
@@ -2,27 +2,34 @@
...
 - Raspberry Pi (the board in a plastic case with many USB ports)
 - Power Board (the board in a plastic case with many green sockets and two buttons)
 - Motor Board (the other board in a plastic case)
 - Arduino (The blue, exposed board with a metal USB-B connector)
 - Arduino Servo Shield (another blue board, hopefully on top of the first)
+If any of this is unsure to you check on the underside of the board it should be labelled
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: I've got no idea what this sentence says or what it's actually for?

File: content/tutorials/kit-assembly.md [hunk 3]
```diff
@@ -2,27 +2,34 @@
...
 - Power Board (the board in a plastic case with many green sockets and two buttons)
 - Motor Board (the other board in a plastic case)
 - Arduino (The blue, exposed board with a metal USB-B connector)
 - Arduino Servo Shield (another blue board, hopefully on top of the first)
+If any of this is unsure to you check on the underside of the board it should be labelled 
+- Battery
+- USB Hub
+- USB Webcam
+
+##Connectors and Cables##
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: Remove hashes again

File: content/tutorials/kit-assembly.md [hunk 4]
```diff
@@ -2,27 +2,34 @@
...
+
+##Tools you'll need (Not Supplied)##
 - Pliers
 - Wire Cutters
 - Wire Strippers
-- Screwdriver
+- Screwdriver (2mm)
 - 2 x Motor
-- Anything else will have to be obtained elsewhere… ask someone!
+- Obtain any other needed tool/supplies yourself
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: _tool_ should be pluralised

File: content/tutorials/kit-assembly.md [hunk 5]
```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
 
 - Colour coding is key; try to make sure that you use RED for wires connected to a powered rail (say 12V or 5V), BLACK for wires connected to ground (0V rail) and BLUE for motors, although it is permissible to use other colours in this case if they were already soldered onto the motors provided in the kits.
 
+- DO NOT TOUCH THE MICRO SD CARD IN THE RASPBERRY PI
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: I'm not sure this warning is necessary. If we don't bring attention to it, they _shouldnt_ try and do it for themselves

File: content/tutorials/kit-assembly.md [hunk 6]
```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
...
 2. Connect the USB hub to the Pi by plugging it into any one of its four USB sockets.
 3. Connect the Power Board to the Pi via one of the black micro-usb cables; the standard USB end goes into any USB socket on the Pi or connected USB hub, the micro-usb end into the Power Board.
-4. Connect the Motor Board to the Power Board by screwing the two large (7.5mm) camcons provided onto the opposite ends of a pair of wires, ensuring that positive connects to positive and ground to ground, and then plugging one end into the appropriate socket of the Motor Board and the other into a high power socket on the side of the Power Board.
+4. Connect the Motor Board to the Power Board by screwing the two large (7.5mm) camcons provided onto the opposite ends of a pair of wires, ensuring that positive connects to positive and ground to ground, and then plugging one end into the appropriate socket of the Motor Board (the connections labelled M0 or M1) and the other into a high power socket on the side of the Power Board ().
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: Probably want to remove the brackets on the end of this sentence?

File: content/tutorials/kit-assembly.md [hunk 7]
```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
...
 5. Connect the Motor Board to the Pi by way of the other black micro-usb cable; the standard USB end goes into any USB socket on the Pi or connected USB hub, the micro-usb end goes into the Motor Board.
 6. The Arduino Servo Shield should already be connected into the top of the Arduino, but, if not, the pins on the bottom fit directly into the sockets with the same labels on the top of the Arduino.
-7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
+7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B (this is the same cable the connects to a printer) end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: The additional bracketed bit really isn't necessary. If people don't know what one looks like, maybe linking to an image in the original list would be better?

File: content/tutorials/kit-assembly.md [hunk 8]
```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
...
 8. Connect the Arduino Servo Shield to the Power Board by screwing the green connector on the top and the small (3.81mm) camcon provided onto the opposite ends of a pair of wires and plugging the camcon into the second of the 5V power rail sockets on the Power Board.
 9. Connect the USB Webcam to the Pi via any one of the remaining USB sockets on the Pi or connected USB hub.
 10. To connect the motors, first screw wires connected to their pins into one of the medium-sized (5mm) camcons and plug these into either of the two appropriate sockets on a Motor Board.
-11. At this point, check that everything is connected up correctly and ask someone in a purple Smallpeice T-shirt to double-check then we will give you your battery. You don’t want the battery to blow up your kit!
+11. At this point, check that everything is connected up correctly (it may be helpful to ask someone around you to check that all cables are connected properly, ground to ground, positive to positive).
```
Comment (RealOrangeOne) [inline content/tutorials/kit-assembly.md]: Probably dont need the explicit _"ground to ground, positive to positive"_ here.

File: content/tutorials/kit-assembly.md (hunk 2 above)
Comment (antonnikitin97) [reply content/tutorials/kit-assembly.md]: I think it's saying there's information on the back of the motor/power board that would help them?

**Pull request merged**